src/Deploy/src/configparser.cpp

```diff
@@ -708,7 +708,7 @@ bool ConfigParser::parseDeployMode(bool checkBin) {
         QuasarAppUtils::Params::log("Failed to initialize targets or extra data!",
                                     QuasarAppUtils::Error);
 
-        QuasarAppUtils::Params::log("Use bin or extraData optins. And check input pathes.",
+        QuasarAppUtils::Params::log("Use bin or extraData options. And check input pathes.",
```

Comment:
```suggestion
        QuasarAppUtils::Params::log("Use bin or extraData options and check input paths.",
```

https://www.dictionary.com/browse/path